src/components/periodic-table/PeriodicTable.tsx

```diff
@@ -1,177 +1,101 @@
-import autobind from "autobind-decorator";
 import * as React from "react";
 import { Spinner } from "../shared/spinner/Spinner";
 
 import "./PeriodicTable.scss";
 
-import data from "../../data/pt.json";
-
-export interface IPeriodicTableElement {
-  component: React.ComponentClass;
-  props?: {
-    [key: string]: any;
-  };
-}
+import periodicTableData from "../../data/pt.json";
```

Comment:
This is not necessarily for this PR, but I think it would be nice to have a `ElementsContext` and a `useElements` custom hook for accessing them.  Less dependency on globals and easier injection for tests.

Reply:
I was thinking the same thing. 
A placeholder hook can be introduced for now which initializes the `ElementManager` and provides the data, which can later be swapped out with context.

Reply:
This is an interesting idea. `ElementManager` can be replaced with and hook. It will be super !! :)